packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/DefaultValueField.tsx

```diff
@@ -102,7 +111,7 @@ function DefaultValueListField(
         // if it's a DATE_LIST, then the values are Unix timestamps in milliseconds
         // The `MultiValueField` component expects ISO strings to match the JSON Schema 'date-time' format
         return _value.map(v => {
-          return dayjs(v).toISOString()
+          return dayjs(v, 'x').toISOString()
```

Comment:
No bug here since we parse the integers at a higher level, but this makes it a little safer